.github/workflows/e2e-container.yaml

```diff
@@ -0,0 +1,37 @@
+name: End to end tests - containerd
```

Comment:
`e2e tests...` instead ?

Reply:
I plan on adding another e2e test target for cri-o runtimes. Hence I need to differentiate them.

Reply:
`e2e tests - containerd`
:)

Reply:
thanks, just because e2e is a term that we are used to, takes less infinitesimal "effort" to read it